docs/get_started/setup.md

```diff
@@ -112,55 +112,66 @@ These commands on Windows will set the environment variable for the duration of
+- Download [Ollama](https://ollama.com/download).
+- After setting up Ollama, pull the Llama3 model by typing the following command into the terminal:
+```bash
+ollama pull llama3
+```
+- Create a ModelFile similar the one below in your project directory.
+```bash
+FROM llama3
 
-    # Launch the model worker
-    python -m fastchat.serve.model_worker \
-        --model-path meta-llama/Llama-2-7b-chat-hf  # a local folder or HuggingFace repo Name
+# Set parameters
+PARAMETER temperature 0.8
+PARAMETER stop Result
 
-    # Launch the API server
-    python -m fastchat.serve.openai_api_server \
-        --host localhost \
-        --port 8000
-    ```
+# Sets a custom system message to specify the behavior of the chat assistant
 
+# Leaving it blank for now.
 
-2. Initialize the agent.
-    ```python
-    # Import the necessary classes
-    from camel.configs import ChatGPTConfig, OpenSourceConfig
-    from camel.types import ModelType, ModelPlatformType
-    from camel.models import ModelFactory
-
-    # Set the LLM model type and model config
-    model_platform = ModelPlatformType.OPENSOURCE
-    model_type=ModelType.LLAMA_2                     # Specify the model type
-    model_config=OpenSourceConfig(
-        model_path='meta-llama/Llama-2-7b-chat-hf',  # a local folder or HuggingFace repo Name
-        server_url='http://localhost:8000/v1')      # The url with the set port number
-
-    # Create the backend model
-    model = ModelFactory.create(
-        model_platform=model_platform,
-        model_type=model_type,
-        model_config=model_config)
-
-    # Set the arguments
-    agent_kwargs = dict(
-        model=model,
-        token_limit=2046,                                # [Optional] Choose the ideal limit
-    )
-
-    # Now your agent is ready to play
-    agent = ChatAgent(sys_msg, **agent_kwargs)
-    ```
+SYSTEM """ """
+```
+- Create a script to get the base model (llama3) and create a custom model using the ModelFile above. Save this as a .sh file:
+```bash
+#!/bin/zsh
+
+# variables
+model_name="llama3"
+custom_model_name="camel-llama3"
+
+#get the base model
+ollama pull $model_name
+
+#create the model file
+ollama create $custom_model_name -f ./Llama3ModelFile
+```
+- Navigate to the directory where the script and ModelFile are located and run the script. Enjoy your Llama3 model, enhanced by CAMEL's excellent agents.
+```python
+from camel.agents import ChatAgent
+from camel.messages import BaseMessage
+from camel.models import ModelFactory
+from camel.types import ModelPlatformType
+
+ollama_model = ModelFactory.create(
+    model_platform=ModelPlatformType.OLLAMA,
+    model_type="llama3",
+    url="http://localhost:11434/v1",
+    model_config_dict={"temperature": 0.4},
+)
+
+assistant_sys_msg = BaseMessage.make_assistant_message(
+    role_name="Assistant",
+    content="You are a helpful assistant.",
+)
+agent = ChatAgent(assistant_sys_msg, model=ollama_model, token_limit=4096)
```

Comment:
`model` -> `model_type`

Reply:
`ollama_model = ModelFactory.create(
    model_platform=ModelPlatformType.OLLAMA,
    model_type="camelai-llama3",
    url="http://localhost:11434/v1",
    model_config_dict={"temperature": 0.4},
)`

Hey @lightaime , since `model_type` was used to set the model using `ModelFactory`, so I used `model` instead of `model_type` to avoid confusion, WDYT? Should we still use `model_type` for `ChatAgent`?